packages/split-view/stories/split-view.stories.ts

```diff
@@ -0,0 +1,205 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { html, TemplateResult } from '@spectrum-web-components/base';
+
+import '../sp-split-view.js';
+
+export default {
+    title: 'Split View',
+    component: 'sp-split-view',
+};
+
+export const Horizontal = (): TemplateResult => {
+    return html`
+        <sp-split-view style="height: 200px" primary-default="100">
+            <div slot="primary">First panel</div>
+            <div slot="secondary">Second panel</div>
+        </sp-split-view>
+    `;
+};
+
+export const HorizontalResizable = (): TemplateResult => {
+    return html`
+        <sp-split-view
+            resizable
+            primary-min="50"
+            primary-max="Infinity"
+            secondary-min="50"
+            secondary-max="Infinity"
+        >
+            <div slot="primary">
+                <h1>First panel</h1>
+                <p>
+                    Lorem Ipsum is simply dummy text of the printing and
+                    typesetting industry.
+                </p>
+            </div>
+            <div slot="secondary">
+                <h2>Second panel</h2>
+                <p>
+                    It is a long established fact that a reader will be
+                    distracted by the readable content of a page when looking at
+                    its layout.
+                </p>
+            </div>
+        </sp-split-view>
+    `;
+};
+
+export const HorizontalResizableCollapsible = (): TemplateResult => {
+    return html`
+        <sp-split-view
+            resizable
+            collapsible
+            primary-min="50"
```

Comment:
Using these two values together also prevents the keyboard interface from reaching `collapsed`. Should we just skip from the `min` to `collapsed` in that case?

Reply:
resolved this now by removing the `collapsible` flag.